scripts/env/dev/init.sh

```diff
@@ -52,14 +52,31 @@ check_supported_kernel() {
 
 #function to install/upgrade cortx-py-utils rpm
 install_cortx_py_utils() {
-  #rpm -q cortx-py-utils && yum remove cortx-py-utils -y && yum install cortx-py-utils -y
+  #install cpio
+  if rpm -q 'cpio' ; then
+    echo "cpio already present ... Skipping ..."
+  else
+    yum install cpio -y
+  fi
+
+  # download cortx-py-utils.
+  yumdownloader --destdir="$PWD" cortx-py-utils
```

Comment:
please do the check and cleanup of stale data of previous run from "$PWD" directory before we download py-utils pkg.
there is possibility of stale data of previous run if we see any failures in following step

Reply:
Fixed.
shifted cleanup from after installation to before installation so that it can handle failure scenario.